presentation/src/main/java/com/example/dms_android/feature/mypage/CheckOriginPwScreen.kt

```diff
@@ -81,7 +81,7 @@ fun CheckPwValue() {
     )
     DormTextField(
         modifier = Modifier
-            .padding(start = 24.dp, end = 24.dp) ,
+            .padding(start = 24.dp, end = 24.dp),
```

Comment:
```suggestion
            .padding(horizontal = 24.dp),
```
이렇게 작성하시면 좀 더 깔끔해 질 것 같아요.

Reply:
아 이거 제가 짠 코드 아니에요. 왜 이게 여깄지?

Reply:
용진이 영역 같은데